README.md

```diff
@@ -0,0 +1,39 @@
+# Shardeum Network Status
+
+This repo is developed to check for the Status of the Shardeum Network services.
+
+This is to ensure whether the services are running are turned off.
+
+### Getting Started.
+To get started, ensure you have `Node.js` installed preferably v18+ 
+
```

Comment:
You can also remind developers to adhere to the requirements of each submodule in the repo. e.g:

```suggestion
Adhere to the requirements of each submodule (frontend & backend)
```